contracts/modules/cover/Gateway.sol

```diff
@@ -161,6 +164,12 @@ contract Gateway is MasterAware {
     return claimId;
   }
 
+  function claimTokens(uint coverId, uint incidentId, uint coveredTokenAmount)
```

Comment:
We need to proxy the tokens through the Gateway contract. Even though it will take slightly more gas - the gateway contract idea is to be the only interface an integrator would use. If we're not proxying the token transfer via this contract they'd have to approve the Incidents contract instead of the gateway which is confusing.